### DISCUSSION TITLE
Is it possible to disable full /home sharing?

### DISCUSSION THREAD

**Discussion by odomingao** [winapps-org/winapps#271] [Support]
>The GNU/Linux /home directory is accessible within Windows via the \\tsclient\home mount.

Is it possible to disable this, and instead set up a separate shared folder?

**Comment by TheBolshe** [comment]
I would like this too, especially since the default config already mounts /home inside the data shared folder.

**Comment by gitnohubz** [comment]
@oskardotglobal i cant see the reason here

**Comment by kanchudeep** [comment]
Full user (home) directory sharing can be disabled by editing the `winapps` script at `~/.local/bin/winapps`.

Edit the file & completely remove all lines passing the argument '`+home-drive \`' to the RDP client (lines 526, 545, 582, 608 in the current version).

**Reply by odomingao** [reply]
Didn't expect to get an answer after all this time, thanks! I'll try.
